health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -54,6 +55,132 @@ export const UICustomizations = {
+                      tenantId: tenantId ,
+                      type: "boundaryManagement",
+                      hierarchyType: row?.hierarchyType,
+                      campaignId: "default"
+                  }
+              });
+              return res;
+            }
+            const downloadExcelTemplate = async() => {
+              const res = await generateFile();
+              const resFile = await generateTemplate();
+              if (resFile && resFile?.GeneratedResource?.[0]?.fileStoreid) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)